src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxSqlUtils.java

```diff
@@ -516,7 +516,7 @@ static String getDocNotesSelectSql(DocumentNoteRow docNoteRow) {
         appendQuery(selectSql,
                 SqlText.SELECT, docNoteRow.orderedFields, SqlText.FROM, docNoteRow.tables,
                 SqlText.WHERE,
-                        docNoteRow.remoteObjectIdentifier, SqlText.EQUALS, "(SELECT OBJ_ID FROM KRNS_DOC_HDR_T WHERE DOC_HDR_ID = ?)");
+                        docNoteRow.remoteObjectIdentifier, SqlText.EQUALS, "(SELECT OBJ_ID FROM FS_DOC_HEADER_T WHERE FDOC_NBR = ?)");

```

Comment:
Does the SQL need a schema reference? (kfs.)

Reply:
It ran fine without it, but I can add it and then we can validate further in DEV.

Reply:
okay given that the schema reference wasn't there before I will merge.

Reply:
Schema reference has been added. Further cleanup of schema references in this process will be handled in separate tickets as appropriate.